### ISSUE TITLE
Scrolling signal on RichTextLabel's scrollbar doesn't work when scrolling with scroll wheel

### ISSUE THREAD

**Issue by ShatReal** [godotengine/godot#46421]
**Godot version:**

3.2.4-rc3 (This is also an issue on 3.2.3).

**OS/device including version:**

Windows 10

**Issue description:**

The `scrolling` signal on a RichTextLabel's scrollbar will not emit when it's scrolled with the scroll wheel. Clicking and dragging it still works.

**Steps to reproduce:**
1. Make RichTextLabel
2. Add enough text to trigger the scrollbar to appear
3. Connect scrollbar's `scrolling` signal
4. Scroll with scroll wheel

**Minimal reproduction project:**

[test.zip](https://github.com/godotengine/godot/files/6043821/test.zip)

**Comment by Calinou** [comment]
Does it work with a ScrollContainer?

**Comment by ShatReal** [comment]
No, it's not emitted when using the mouse wheel on a ScrollContainer as well.

**Comment by SeleDreams** [comment]
I can confirm this issue is present on 3.3.2 as well for me

**Comment by IsotoxalDev** [comment]
this issue still persist in 3.4.4

**Comment by autofool** [comment]
Can confirm on 3.5.1 stable mono.

**Comment by mhilbrunner** [comment]
Can confirm this is still current as of Godot 4.0.2, with the behaviour mentioned above:
- Scrolling with keyboard buttons or mouse wheel **does not** trigger signals
- Grabbing the scroller and moving it **does** trigger signals

**Comment by t-karcher** [comment]
> "scroll_started" and "scroll_ended" signals of scroll container does not work at all, no matter which child is used (Control, Label, Rect, etc.). While "scrolling" signal of scrollbar only works when scroll handle is grabbed.

This is a separate bug, already reported in #22936

**Comment by TheSofox** [comment]
It occurred to me that fast fix would just be to have "scrolling" fired whenever "_value_changed" is called, but that might be too broad.
Could I have some more direction for when "scrolling" is meant to be fired? I assume if its scrolled with keyboard or mouse then it should be fired, but if it's moved programmatically it shouldn't? The documentation just says "Emitted when the scrollbar is being scrolled."
